rom/dev/README.md

```diff
@@ -397,6 +400,29 @@ The following is the sequence of the steps that are required to download the par
 
 ![DATA FROM MBOX FLOW](doc/svg/data-from-mbox.svg)
 
+Following is the sequence of steps that are performed to download the firmware image into the mailbox in ACTIVE mode.
+
+- On receiving the RI_DOWNLOAD_FIRMWARE mailbox command, set Recovery Interface (aka RI) PROT_CAP register [Byte11:Bit3] to 1 ('Flashless boot').
+- Set the RI DEVICE_STATUS register Byte0 to 0x3 ('Recovery mode - ready to accept recovery image').
+- Set the RI DEVICE_STATUS register Byte[2:3] to 0x12 ('Recovery Reason Codes' 0x12 = 0 Flashless/Streaming Boot (FSB)).
+- Set the RI RECOVERY_STATUS register [Byte0:Bit[3:0]] to 0x1 ('Awaiting recovery image') and [Byte0:Bit[7:4]] to 0 (Recovery image index).
+- Loop on the 'payload_available' signal for the firmware image details to be available.
+- Read the image size from RI INDIRECT_FIFO_CTRL register Byte[2:5]. Image size in DWORDs.
```

Comment:
Note to self. It looks like emulator needs to be updated for this as some alignment was assumed. there